pkg/spec/spec.go

```diff
@@ -399,6 +403,13 @@ func setupCapabilities(config *CreateConfig, configSpec *spec.Spec) error {
 	configSpec.Process.Capabilities.Permitted = caplist
 	configSpec.Process.Capabilities.Inheritable = caplist
 	configSpec.Process.Capabilities.Effective = caplist
+	if config.User != "" {
```

Comment:
Probably worth doing a quick check here to see that User is not "0" or "root" - we'll want to keep caps if it is.